src/index.html

```diff
@@ -5,8 +5,11 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <script type="text/javascript" src="app.js"></script>
-    <script type="text/javascript" src="../node_modules/paper/dist/paper-full.js"></script>
+    <!--<script type="text/javascript" src="../node_modules/paper/dist/paper-full.js"></script>-->
+    <script type="text/javascript" src="libs/paper-full.js"></script>
```

Comment:
This is done on this way because we use a `paper.js` version from the `develop` branch and it's not available through NPM